dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -93,7 +93,10 @@ public Optional<List<PagedSearchIterable<GHContent>>> findFilesWithImage(
         // Filename search appears to yield better / more results than language:Dockerfile
         // Root cause: linguist doesn't currently deal with prefixes of files:
         // https://github.com/github/linguist/issues/4566
+        // This will work in OR mode i.e, either filename is Dockerfile or docker-compose
         search.filename("Dockerfile");
+        search.filename("docker-compose");
```

Comment:
Can we move these filenames to come from CLI where we have a default list of `"Dockerfile", "docker-compose"` there? Then this part of code would do search.filename on a list

Reply:
Done